installer/PowerToysSetup/Product.wxs

```diff
@@ -1013,6 +1018,36 @@
         <?endforeach?>
       </Component>
 
+      <!-- Uri Plugin -->
+      <Component Id="UriComponent" Directory="UriPluginFolder" Guid="C7DC8F88-554C-4375-9510-9435399B5D3D">
+        <?foreach File in plugin.json;Wox.Infrastructure.dll;Wox.Plugin.dll;Microsoft.Plugin.Uri.deps.json;Microsoft.Plugin.Uri.dll;Telemetry.dll?>
+          <File Id="Uri_$(var.File)" Source="$(var.BinX64Dir)modules\launcher\Plugins\Microsoft.Plugin.Uri\$(var.File)" />
+        <?endforeach?>
+      </Component>
+      <Component Id="UriImagesComponent" Directory="UriImagesFolder" Guid="8C9C1634-28C8-45C4-A8EA-8D4C9B4810D0">
```

Comment:
@royvou I think you want this to be `UriLanguagesFolder`.  I see the following when running from the installer. 

![image](https://user-images.githubusercontent.com/56318517/89677248-f336a380-d8a1-11ea-9ea0-f75667d2fe1f.png)

Reply:
Sorry the comment is regarding this element `<Component Id="UriLanguagesComponent" Directory="UriImagesFolder"`

Reply:
I'll double check this, looks like a small find/replace error!